app/Http/Controllers/Api/ContactController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+use App\Domains\Contacts\Contact;
+
+class ContactController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return Contact::all();
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        if (!Contact::create($request->all())) {
```

Comment:
It doesn't work this way with laravel.

You should validate every input of the request so a proper record can be created, a 422 error should thrown if anything goes wrong.

Error 500 will be called automatically by laravel whenever an exception happens.